lib/pcb/convert-circuit-json-to-pcb-svg.ts

```diff
@@ -248,6 +257,8 @@ function createSvgObjects(
       return createSvgObjectsFromPcbHole(elm, transform)
     case "pcb_smtpad":
       return createSvgObjectsFromSmtPad(elm, transform)
+    case "pcb_solder_paste":
+      return createSvgObjectsFromSolderPaste(elm, transform)
```

Comment:
we never want to draw the solder paste for pcb svgs

also the way you implemented `Options` here is not the tscircuit 2 parameter convention https://github.com/tscircuit/handbook/blob/main/guides/code.md#1-two-parameter-rule